tests/queries/0_stateless/02932_analyzer_rewrite_sum_column_and_constant.sql

```diff
@@ -0,0 +1,208 @@
+-- { echoOn }
```

Comment:
You could have set it only once at the top of the file:
```
SET allow_experimental_analyzer=1;
```
But all is good. Nice and comprehensive set to tests

Reply:
Thanks for your review. I will do that at the top of test.

Reply:
Great. Thanks a lot